packages/ui-components/src/components/Map/MarkerLayer.js

```diff
@@ -19,13 +19,9 @@ const ShadedPolygon = styled(Polygon)`
   }
 `;
 
-const getText = (measure, serieses) => {
-  const { name } = measure;
+const getText = ({ name, ...measure }, serieses) => {
   const hasMeasureValue = measure || measure === 0;
```

Comment:
I don't understand how `measure === 0` could be true here with the destructuring that's happening

Reply:
I copied that across from tupaia to try and not miss any edge cases but it definitely doesn't make sense anymore. I tried debugging it on web-frontend to figure out what it's there for but couldn't find any examples of when it's needed.